collections/protocodec/go.mod

```diff
@@ -0,0 +1,54 @@
+module cosmossdk.io/collections/protocodec
```

Comment:
Why is this its own go.mod? I feel like it should just be a package.

Reply:
talked in slack